docs/contracts.md

```diff
@@ -47,9 +47,8 @@ output = PublicOut(Sec, Pub)
 ```
 
 Noninterference essentially demands that for any two secrets `Sec1` and `Sec2` and any public input `Pub`, the program’s behavior from an attacker’s perspective is identical when run on `(Sec1, Pub)` versus `(Sec2, Pub)`:
-
 <center>
-&nbsp;&nbsp;&nbsp;**Definition 1 (Noninterference)**: A program `P` is noninterferent if, for all public inputs `Pub` and all secret inputs `Sec1`, `Sec2`, if `Sec1 = Sec2`, then `PublicOut(P, Sec1, Pub) = PublicOut(P, Sec2, Pub)`.
+&nbsp;&nbsp;&nbsp;**Definition 1 (Noninterference)**: A program `P` is noninterferent if, for all<br>public inputs `Pub` and all pair of secret inputs `Sec1`, `Sec2` we have that <br>`PublicOut(P, Sec1, Pub) = PublicOut(P, Sec2, Pub)`.
```

Comment:
1. pairS of secret inputs
2. "we have that" sounds out of place here

Reply:
Maybe we should actually make this definition similar to Def 2, namely use a pair of Pub inputs instead of just one. This should make the idea more clear + will highlight the similarity between the definitions

Reply:
> namely use a pair of Pub inputs instead of just one

What about changing definition 2 to have only one `Pub` value? Using `Pub1 = Pub2` is nearer to the classic formal definition, but using only one `Pub` value might make it clearer that it's the same exact public input.